samples/sampler/stories/default/PageViews.js

```diff
@@ -17,7 +18,7 @@ export default {
 };
 
 export const _PageViews = (args) => (
-	<PageViews fullContents={args['fullContents']} pageIndicatorType={args['pageIndicatorType']}>
+	<PageViews css={css} fullContents={args['fullContents']} pageIndicatorType={args['pageIndicatorType']}>
```

Comment:
Why do we need this change in default story?
Besides, the css has no specific overrides.
Generally, we don't add this to the default stories. I think it's ok not to add this.

Reply:
It's fixed.